pkgs/servers/web-apps/plausible/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib, stdenv, beamPackages, fetchFromGitHub, glibcLocales, cacert
```

Comment:
really a nitpick, but would you mind running the nixpkgs-fmt formatter.
Just so there is consistent formatting